### ISSUE TITLE
Database dies at playing multiple titles  on BubbleUPnP

### ISSUE THREAD

**Issue by udippel** [gerbera/gerbera#2856]
BubbleUPnP as client, select random play, accept the default 10 titles, 10 titles are displayed, database crashes. 
From then on nothing can be played and the web interface states
![image](https://github.com/gerbera/gerbera/assets/50265752/eb9f63d6-e6dc-445a-8288-2deb03004ce8)
gerbera is still running, and it is still possible to login, but that's the statement after login.

(Of course I am willing to help to improve gerbera, since it has a lot of potential, and is overall nicely designed. Alas, my coding as of today is next to zero. Though when a new version with bug fixes comes out (as .deb), I will gladly try to continue testing.)

**Comment by udippel** [comment]
This is where the database dies reproducibly:
```
2023-08-14 10:35:39  error: Exception: [/build/src/database/sqlite3/sqlite_database.cc:313] virtual int Sqlite3Database::exec(const string&, bool) Error: [/build/src/database/sqlite3/sl_task.cc:61] void SLTask::waitForTask() Error: SQLITE3 (19: 1555): UNIQUE constraint failed: grb_playstatus.group, grb_playstatus.item_id
       Query: INSERT INTO "grb_playstatus" ("group","item_id","playCount","lastPlayed","lastPlayedPosition","bookMarkPos") VALUES ('default',208082,8,1692002139,0,0)
       Error: UNIQUE constraint failed: grb_playstatus.group, grb_playstatus.item_id
2023-08-14 10:35:39  error: prematurely shutting down.
[/build/src/database/sqlite3/sqlite_database.cc:407] void Sqlite3Database::addTask(const std::shared_ptr<SLTask>&, bool) Error: sqlite3 task queue is already closed
```

**Comment by udippel** [comment]
So far no (re)action. Therefore I continued trying other players. Gerbera seems to do fine with Kodi so far.(Kodi DOES have a silly interface, beware).
Since my much preferred BubbleUPnP, mentioned on top in Gerberas documentation as well, seems to make the darabase fall over regularly, the question arises, on which side the problem could originate?
Could it make sense to file a bug against BubbleUPnP?

**Comment by KarlStraussberger** [comment]
Sorry, but the database dies because of too many requests. The problem is muli-threading of server calls. Obviously bubble triggers too many command at the same time. This is hard to reproduce because it depends on hardware caps and os versions.

**Comment by KarlStraussberger** [comment]
Sorry, but we're not a professional organisation here. We're doing this in the time our regular jobs do not occupy us and our families can spare us.

As soon as somebody comes up with a PR containing fixes for an open issue it is discussed and integrated into the code.

I know there are some issues in the sqlite interface. If you want to contribute to the project, feel free to build the current HEAD and test whether the issue is still there or already fixed in the wake of other changes made. Then we would also have the hard- and software combination that leads to the issue.

**Comment by udippel** [comment]
I know, I did FOSS development myself, starting in 1998. My intention was not to point fingers at anyone. I do appreciate voluntary work very much.
My intention was, and remains, to improve FOSS projects. If only, to create the awareness of certain difficulties, including to developers. 
Minimum here would be, to help others encountering the same problem to understand better potential reasons for their observed troubles, pointing to possible workarounds.
Like the other one that I had opened: If only the documentation had stated clearly 'Do Not Update!' instead of "We strongly advise using the packages provided by your distribution if available", it would have saved plenty of everyone's time. Likewise here: listing out BubbleUPnP on top of 'supported clients' seems less than optimal. 

Thanks again, and yes, of course I appreciate your work, and your help!
